components/prism-qore.js

```diff
@@ -0,0 +1,20 @@
+Prism.languages.qore = Prism.languages.extend('clike', {
+  'comment': {
+    pattern: /(^|[^\\])(\/\*[\w\W]*?\*\/|((\/\/)|#).*?(\r?\n|$))/g, // /(^|[^\\])([\w\W]*?\*\/|(^|[^:])\/\/.*?(\r?\n|$))/g,
```

Comment:
Is it important to consume the line feed ? Otherwise, I think the regexp could end simply with `.+`